proto/tikvpb.proto

```diff
@@ -83,22 +84,23 @@ message BatchCommandsRequest {
             kvrpcpb.BatchRollbackRequest BatchRollback = 8;
             kvrpcpb.ScanLockRequest ScanLock = 9;
             kvrpcpb.ResolveLockRequest ResolveLock = 10;
-            kvrpcpb.GCRequest GC = 11;
-            kvrpcpb.DeleteRangeRequest DeleteRange = 12;
-
-            kvrpcpb.RawGetRequest RawGet = 13;
-            kvrpcpb.RawBatchGetRequest RawBatchGet = 14;
-            kvrpcpb.RawPutRequest RawPut = 15;
-            kvrpcpb.RawBatchPutRequest RawBatchPut = 16;
-            kvrpcpb.RawDeleteRequest RawDelete = 17;
-            kvrpcpb.RawBatchDeleteRequest RawBatchDelete = 18;
-            kvrpcpb.RawScanRequest RawScan = 19;
-            kvrpcpb.RawDeleteRangeRequest RawDeleteRange = 20;
-            kvrpcpb.RawBatchScanRequest RawBatchScan = 21;
-
-            coprocessor.Request Coprocessor = 22;
-            kvrpcpb.PessimisticLockRequest PessimisticLock = 23;
-            kvrpcpb.PessimisticRollbackRequest PessimisticRollback = 24;
+            kvrpcpb.RefreshLockRequest RefreshLock = 11;
+            kvrpcpb.GCRequest GC = 12;
+            kvrpcpb.DeleteRangeRequest DeleteRange = 13;
+
```

Comment:
These IDs are changed. Won't this cause incompatibility?

Reply:
ok, I will append this request to the end of batch commands and maintain the original order.